package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@fastify/otel",
+  "version": "0.0.0",
+  "description": "Official Fastifyt OpenTelemetry Instrumentation",
+  "main": "index.js",
+  "type": "commonjs",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "lint": "standard",
```

Comment:
could we use neostandard?

example: https://github.com/fastify/fastify-helmet/pull/266